packages/core/src/messagingProtocols/mls/EventHandler/events/messageAdd/messageAdd.ts

```diff
@@ -70,9 +68,7 @@ const handleMLSMessageAdd = async ({mlsService, event, logger}: HandleMLSMessage
   }
 
   if (!message) {
-    const newEpoch = await mlsService.getEpoch(groupIdBytes);
-    logger.log(`Received commit message for group "${groupId}". New epoch is "${newEpoch}"`);
-    return;
+    return {event};
```

Comment:
This will be reimplemented in a better way in a follow up PR

Reply:
That's the only change right? The rest is just refactorin I'm not missing anything?

Reply:
No, you are right, the rest is only refactoring :)